packages/vm/core/governance/interface.go

```diff
@@ -47,7 +47,7 @@ var (
 	FuncStopMaintenance      = coreutil.Func("stopMaintenance")
 	ViewGetMaintenanceStatus = coreutil.ViewFunc("getMaintenanceStatus")
 
-	// L1 metadata
+	// L2 metadata
```

Comment:
```suggestion
	// public metadata
```
or something like this